tests/blackbox/testlib/dbus.py

```diff
@@ -466,3 +466,11 @@ def get_report(report_name):
             StratisDbus._REPORT_IFACE,
         )
         return iface.GetReport(report_name, timeout=StratisDbus._TIMEOUT)
+
+    @staticmethod
+    def reconnect():
+        """
+        Close and reopen bus connection.
+        """
+        StratisDbus._BUS.close()
+        StratisDbus._BUS = dbus.SystemBus()
```

Comment:
Since `StratisDbus._BUS` is reset, it is no longer a constant. Should it be renamed?

Reply:
If I recall correctly, it just has to do with the case of the variable (`_BUS` vs `_bus`). Try that and see if pylint is okay with that. I'm not sure how Python will react to variables being mutated in a singleton.